skills_DS/api/views.py

```diff
@@ -158,6 +158,17 @@ def parse_resume_async(v, path, request):
 		for skill in data['skills']:
 			skills.append(skill.strip())
 
+		skillsCurrent = list(Profile.objects.filter(user=request.user).values_list('skills',flat=True))
+		y = json.loads(skillsCurrent[0])
+
+		if(len(y) > 0):
+			for skill in y:
+				if skill not in skills:
+					skills.append(skill)
+
+		if '' in skills:
+			skills.remove('')
```

Comment:
don't think this works, you should put this on the for loop above to if else empty string or sth similar